site/src/scaffold/scripts/popover.ts

```diff
@@ -0,0 +1,72 @@
+export function initPopovers(container: HTMLElement) {
+	container.addEventListener('click', (event: Event) => {
+		let targetPopover: HTMLDetailsElement | null;
+		targetPopover =
+			event.target instanceof Element &&
```

Comment:
Mind explaining why we're using `Element` here instead of `HTMLElement`?

Reply:
Because if there is a SVG icon pasted on the summary (as in one of the examples) the target would be `SVGElement`, so instead of writing here `event.target instanceof HTMLElement || event.target instanceof SVGElement` I decided to go with `event.target instanceof Element`